### ISSUE TITLE
[Discussion] Change "Leader" to "Steward" and "Deputy Leader" to "Deputy Steward"

### ISSUE THREAD

**Issue by XertroV** [voteflux/flux#24]
The spirit of how the Leader position was originally set up was that of a steward - someone to steer Flux and take responsibility for direction, integrity, etc, until it was able to sustain itself.

Sounds like a good change.

Original suggestion: https://twitter.com/PallerJohn/status/967975467591467009

**Comment by sventhebarbarian** [comment]
I agree, I think leader can carry negative connotations and steward might be better. Leader does impart a higher sense of power and responsibility, which can be a good thing when trying to direct a number of supporters, I would say it is up to how Nathan and yourself want to view yourselves and have others view you.

If steward, it might by beneficial to have a rough outline on the expected milestones the movement would need to meet for the stewardship to phase out, otherwise it might just seem like word play?

**Comment by qreeves** [comment]
I have to agree with @sventhebarbarian on all points. It may be beneficial in an organisational sense to maintain more of a "leadership" role than a "guardian", especially while trying to set everything up and make the connections necessary to succeed as a party and as a platform.

If it is nothing else other than word play semantics for the sake of it, then there isn't much of a point. I think what matters to us is that things are done transparently and through a method that we can get involved in - this is possible no matter which choice gets made.

In my experience working on open source projects, you often *need* a leader role in order to make the right decisions because giving everyone a vote in everything is inherently fallible and may not be in the best interests of the project/community. While I maintain an open and transparent process, ultimately there are times where I need to make an "executive decision" for what I see as being the correct way forward. I've found people are often averse to change, even if they have no idea what the consequences of the change would be.

So yeah, you need to decide which option would be in the best interests going forward. I think it is fairly well understood how things will work in the future with IBDD and that the current setup is a shortcut to that end.

**Comment by sventhebarbarian** [comment]
Love how they brush over the direct democracy elements, and focus on the negative, dictatorial position.

I enjoy using the term benevolent dictator but its easy for people to misunderstand its application in this type of system, need to try and drop it out of my vocabulary.

**Comment by sventhebarbarian** [comment]
But yeah its probably not a good term to bandy about in public documents, chum for media sharks, steward would be harder to blindly attack..

I was also thinking about how the terms carry different levels of authority, leader kind of implies that all decisions need to be authorized by that person/s, I feel the term steward might suggest authority is not so concentrated. For example volunteers might not proceed with an action until the leader signs off, but if there is no leader, only a steward, those volunteers might feel comfortable going forward on the acceptance of a group of other supporters, kind of softly shifting authority away from leaders and to the community..

**Comment by XertroV** [comment]
@qreeves Yeah - I'm not allowed to edit the wiki page bc I'm a subject of it. Technically ppl with conflicts of interest shouldn't, but some nice random person out there could definitely edit with a citation saying this is no longer valid 😉 

@sventhebarbarian re wiki bias: yeah it's an issue. I started a talk page a year ago but neglected to follow up with it - you can see that for more details of corrections and things I suggested. If Flux wants to suggest edits we should do it through the talk page and mention the guy who does most of the maintenance.

@sventhebarbarian re steward: yeah - we do want ppl to feel empowered but part of the leadership structure is "protecting the meme" - like it's a good meme but it's not simple, so before we can decentralise further we need to make sure it's instantiated well enough to survive. This is pretty hard, but doable.

**Comment by sventhebarbarian** [comment]
Not confident in my wiki etiquette, but would like to adjust or clarify the benevolent dictator line:

> Flux Party constitutions give the party leader the powers of a "benevolent dictator", with members being unable to vote on party decisions.

to something like 

> Flux Party constitutions give the party leader the powers of a "benevolent dictator", with members unable to vote on the direction of the party, the reason given for this is to ensure the party maintains its goal of delivering Issue Based Direct Democracy.

I would like to remove the reference [3] to the article, I think the information there is glossed over and off point, but should add a reference to somewhere that explains the "reason" given above, and also a link to an wiki page for IBDD.

Not really sure if my wording is up to wiki standards ..

**Comment by XertroV** [comment]
You could just add [citation needed] :P 

Personally I think it should just be removed. With the comment mentioning it's not in the constitution.

**Comment by sventhebarbarian** [comment]
done

**Comment by TomSesselmann** [comment]
@xertrov they’ve used web.archive to cite other stuff on our wiki page, and in this case you could even just point to a previous commit.

**Comment by XertroV** [comment]
Yup, but it's inaccurate to say `Flux Party constitutions give the party leader the powers of a "benevolent dictator",`

Also AFAIK the state constitutions have never done this.

**Comment by XertroV** [comment]
@sventhebarbarian - your revision was undone.

Points:
* It's against wiki policy for you to edit it
* Your edit comment was unrelated and if you were a legitimate Wikipedia contributor you wouldn't have said that, you'd say "Removed outdated information, see: [link]" or something

**Comment by qreeves** [comment]
Yeah, what's going on here they define as "[canvassing](https://en.wikipedia.org/wiki/Wikipedia:Canvassing)" and because you are involved with Flux it is automatically assumed you do no have a [neutral point of view](https://en.wikipedia.org/wiki/Wikipedia:Neutral_point_of_view) which is therefore a [conflict of interest](https://en.wikipedia.org/wiki/Wikipedia:Conflict_of_interest).

Wikipedia is full of trolls just looking for an argument, so it is best not to poke the nest. Trust me, I've been through this already and it resulted in the deletion of our page (which turned into an [absolute mess](https://en.wikipedia.org/wiki/Wikipedia:Articles_for_deletion/Red_Eclipse) very quickly). There's so many conflicting policies they can basically justify anything by picking and choosing which ones they follow.

Basically, all information needs [attribution](https://en.wikipedia.org/wiki/Wikipedia:Attribution) and unfortunately the only real articles out there about Flux and IBDD are either severely lacking or paint the party in a less than favourable light. You can't use links to your own articles either because the information needs to come from [independent sources](https://en.wikipedia.org/wiki/Wikipedia:Identifying_and_using_independent_sources) which also have [notability](https://en.wikipedia.org/wiki/Wikipedia:Notability).

The only real solution is to get more media coverage that provides unbiased and useful information about what the party is and what goals it is setting out to achieve, with a proper look at the structure instead of writing it off in an unflattering way.

**Comment by sventhebarbarian** [comment]
Understandable from many angles, your are right we should probably wait for write-ups from notable sources, then hoard those links for times like this..

**Comment by sventhebarbarian** [comment]
There was a bit of coverage in the altcoin community when Max ran the test, I wonder if any of those articles would have a more honest interpretation of Flux, and be notable enough..
